specification/apimanagement/resource-manager/Microsoft.ApiManagement/stable/2017-03-01/apimversionsets.json

```diff
@@ -0,0 +1,467 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "title": "ApiManagementClient",
+        "description": "Use these REST APIs for performing operations on the ApiVersionSet entity associated with your Azure API Management deployment. Using this entity you create and manage API Version Sets that are used to group APIs for consistent versioning.",
+        "version": "2017-03-01"
+    },
+    "host": "management.azure.com",
+    "schemes": [
+        "https"
+    ],
+    "consumes": [
+        "application/json"
+    ],
+    "produces": [
+        "application/json"
+    ],
+    "security": [
+        {
+            "azure_auth": [
+                "user_impersonation"
+            ]
+        }
+    ],
+    "securityDefinitions": {
+        "azure_auth": {
+            "type": "oauth2",
+            "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+            "flow": "implicit",
+            "description": "Azure Active Directory OAuth2 Flow.",
+            "scopes": {
+                "user_impersonation": "impersonate your user account"
+            }
+        }
+    },
+    "paths": {
+        "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ApiManagement/service/{serviceName}/api-version-sets": {
```

Comment:
@darrelmiller - is this resource type already supported on the service side?

Reply:
The resource type names should be lower camel cased and only contain alphanumeric characters. Which is why I was checking.

Reply:
@ravbhatnagar Are you saying that path segment `api-version-sets` should actually be `apiVersionSets` ?  It is currently implemented as the former, but we could change it and make sure will still support the old value as well so we don't break any existing users.

Reply:
@ravbhatnagar  We can do this rename in our 2018-01-01 version which is what we are building the new .net client against.

Reply:
@darrelmiller - sounds good. we can target this change in the new api-version.